spack/repo/packages/py-pyprecice/package.py

```diff
@@ -14,11 +14,12 @@ class PyPyprecice(PythonPackage):
 
     homepage = "https://precice.org"
     git = "https://github.com/precice/python-bindings.git"
-    url = "https://github.com/precice/python-bindings/archive/v2.0.0.1.tar.gz"
+    url = "https://github.com/precice/python-bindings/archive/v2.4.0.0.tar.gz"
```

Comment:
Just as a note: Normally this does not have to be updated for new releases. Spack needs a url to the tar ball which includes the version numbering such that Spack knows how to subsitute new versions. From there on it will reconstruction/update the URL from the version statements below.

Reply:
Okay, understood :+1: